default/templates/default/package-releases.html

```diff
@@ -7,7 +7,7 @@
 in the box below to create a <a href="https://standard.open-contracting.org/latest/en/schema/release_package/" target="_blank">Release Package</a> {% endblocktrans %}.
 {% blocktrans %}You can also click the <strong>Add a file</strong> link below.{% endblocktrans %}
 {% blocktrans %}Use the <strong>Published Date</strong> option to set the Published Date in the output package.{% endblocktrans %}</p>
-<p>{% blocktrans %}The input files must be JSON files with a single Release each.{% endblocktrans%}</p>
+<p>{% blocktrans %}The input files must be JSON files of Releases and can be JSON Array files of Releases.{% endblocktrans%}</p>
```

Comment:
Can you please change this to "or a JSON Array of Releases"?

Reply:
Please use sentence case, e.g. "or a JSON array of releases".